tests/test_host_check.py

```diff
@@ -486,14 +505,67 @@ def test_plat_specified_check_failing(self, capsys):
 ==================================================================
 !! Host NOT set up correctly for xrd-control-plane !!
 ==================================================================
+"""
+        assert output == cli_output
+        assert exit_code == 1
+
+    def test_plat_specified_check_erroring(self, capsys):
+        """Test a platform check erroring when host-check is run with arguments."""
+        exit_code, output = self.run_host_check(
+            capsys,
+            ["-p", "xrd-control-plane"],
+            erroring_checks=["Kernel version"],
+            failing_checks=["Kernel version"],
+        )
+        cli_output = f"""\
+==============================
+Platform checks - xrd-control-plane
+==============================
+Checks: {CONTROL_PLANE_CHECKS_STR}
+
+==================================================================
+!! One or more checks could not be performed, see errors above !!
+==================================================================
```

Comment:
As discussed in person, this summary should also indicate whether there were any failures (which will then mean a couple of extra testcases are needed).